editor/test/resources/editor_extensions/resource_attributes_project/test.editor_script

```diff
@@ -1,10 +1,10 @@
 local M = {}
 
 local function test_resource(resource_path)
-    local success, exists = pcall(editor.resource_exists, resource_path)
+    local success, attrs = pcall(editor.resource_attributes, resource_path)
     local message
     if success then
-        message = exists and "exists" or "does not exist"
+        message = attrs.exists and ("exists (" .. (attrs.is_file and "file" or "directory") .. ")" ) or "does not exist"
```

Comment:
We should explicitly check the value of `attrs.is_directory` here as well. Maybe even ensure their values are boolean?

Reply:
Okay!